cns/ipampool/monitor.go

```diff
@@ -140,6 +140,7 @@ func (pm *Monitor) Start(ctx context.Context) error {
 				}
 			}
 
+			scaler := nnc.Status.Scaler
```

Comment:
If we don't have NCs we probably don't have a `Status` at all yet, does this matter? Will the pool monitor start respecting the real scaler when it comes in later?

Reply:
Yes, next time we get an NNC we will copy the scaler out again, and I think this zero-value scaler is safe to use